cmd/corald/midware/auth.go

```diff
@@ -1,136 +0,0 @@
-package midware
```

Comment:
Did we remove the dependencies we don't need anymore? govendor can show us this.